src/pdfnorm.jl

```diff
@@ -0,0 +1,50 @@
+"""
+    pdfsquaredL2norm(d::Distribution)
+
+Return the square of the L2 norm of the probability density function ``f(x)`` of the distribution `d`:
+
+```math
+\\big(\\int_{S} f(x)^{2} \\mathrm{d} x \\big)^{1/2}
```

Comment:
It seems you accidentally merged the wrong outdated suggestion :slightly_smiling_face: This was correct before since we agreed to return the square of the L2 norm:
```suggestion
\\int_{S} f(x)^{2} \\mathrm{d} x
```
The more recent (and I think valid) suggestion was to remove the type parameter `T` in the definition for `Gamma`: https://github.com/JuliaStats/Distributions.jl/pull/1321/files#r632790018

Reply:
Whooops